### PR TITLE
live-dashboard: Unit tests

### PR DESCRIPTION
**Pull request by karbmk**
Test case for event checkin redirection to live-dashboard added.

### CODE REVIEW

File: openCurrents/tests.py [hunk 1]
```diff
@@ -1,3 +1,94 @@
...
+            'username': 'testuser',
+            'password': 'secret'}
+        self.user = User.objects.create_user(**self.credentials)
+
+    def test_event_checkin(self):
+        """TestCase for successful checkin"""
+        # send login data
+        self.client.login(username='testuser', password='secret')
+        form_data = { 'userid':2,'checkin':True }
+        request = self.client.post('/event_checkin/1/', form_data, follow=True)
```
Comment (nickolashe) [inline openCurrents/tests.py]: @karbmk What's the meaning of the `follow` param, I am curious?

File: openCurrents/tests.py [hunk 2]
```diff
@@ -1,3 +1,94 @@
...
+        #request.user = self.user
+        #pk = '1'
+        #response = event_checkin(request,pk=pk)
+        #self.assertEqual(response.status_code, 200)
```
Comment (nickolashe) [inline openCurrents/tests.py]: @karbmk Any particular reason for commenting this out? Did this not pass?

File: openCurrents/tests.py [hunk 3]
```diff
@@ -1,3 +1,94 @@
...
+    """TestSuite for event checkin"""
+    def setUp(self):
+        """Setting up the user to test event checkin"""
+        self.factory = RequestFactory()
+        self.credentials = {
+            'username': 'testuser',
+            'password': 'secret'}
+        self.user = User.objects.create_user(**self.credentials)
+
+    def test_event_checkin(self):
```
Comment (nickolashe) [inline openCurrents/tests.py]: @karbmk I think we should also assert the corresponding record in UserTimeLog table was created

**Comment by nickolashe** [comment]
@karthik making sure you are seeing my comments on the PR

File: openCurrents/tests.py (hunk 3 above)
Comment (karbmk) [inline openCurrents/tests.py]: Hi, sorry for the late reply. I will take a look on it for sure

File: openCurrents/tests.py (hunk 1 above)
Comment (karbmk) [inline openCurrents/tests.py]: follow parameter is basically to follow the assertions after we hit the url like is the status code 401 or is the page returning a message etc..

File: openCurrents/tests.py (hunk 2 above)
Comment (karbmk) [inline openCurrents/tests.py]: It did fail because the assertEqual did'nt work. I will take a look on that

**Comment by karthik** [comment]
@nickolashe I am not the person you are looking for.

**Pull request closed without merging**